src/components/layout/header/Actions/ToggleEditMode/ToggleEditMode.tsx

```diff
@@ -44,8 +46,25 @@ export const ToggleEditModeAction = () => {
         }),
         radius: 'md',
         id: 'toggle-edit-mode',
-        autoClose: false,
-        title: <Title order={4}>{t('popover.title')}</Title>,
+        autoClose: 10000,
+        title: (
+          <Title order={4}>
+            <Trans
+              i18nKey="layout/header/actions/toggle-edit-mode:popover.title"
+              values={{ size: translatedSize }}
+              components={{
+                1: (
+                  <Text
```

Comment:
Even though the text is underlined, it is not clear that you can click it.
Maybe change the design so it's better visible

Reply:
Update: Can't be improved, should be understandable enough as it is